src/com/twu/biblioteca/Printer.java

```diff
@@ -65,4 +67,16 @@ private static void printLibraryItems(ArrayList<? extends LibraryItem> libraryIt
         }
         System.out.println();
     }
+
+    public static void askMovieName() {
+        System.out.println("Type the name of the movie you want to checkout");
+    }
+
+    public static void notifyUserSuccessfulMovieCheckout() {
+        System.out.println("Thank you! Enjoy the movie");
+    }
+
+    public static void notifyUnsuccessfulMovieCheckout() {
```

Comment:
implent a method `notifyUnsuccessfulCheckout(String element)` instead of having two duplicate methods.